contrib/one_shot_models/graph_topology.py

```diff
@@ -11,10 +11,9 @@
 import warnings
```

Comment:
Let's move this file back to `models/` since we're bringing it back into the main repo